### ISSUE TITLE
[Errno 2] No such file or directory: 'ffmpeg'

### ISSUE THREAD

**Issue by maxpzn** [C0untFloyd/roop-unleashed#449]
[Errno 2] No such file or directory: 'ffmpeg'

Got this Error on Mac M2 Sonoma 14.2 (23C64)

Tried to manually install ffmpeg, but nothing worked.

**To Reproduce**

Start swapping on Mac Machine. 

**Details**
What OS are you using?
- [ ] Linux
- [ ] Linux in WSL
- [ ] Windows 
- [x] Mac

Are you using a GPU?
- [ ] No. CPU FTW
- [ ] NVIDIA
- [ ] AMD
- [ ] Intel
- [x] Mac

**Which version of roop unleashed are you using?**

3.5.1

**Screenshots**
![Bildschirmfoto 2024-02-19 um 23 29 43](https://github.com/C0untFloyd/roop-unleashed/assets/160548435/59dde1fe-8690-4e1f-a8a0-6b8e3f5df476)

**Comment by C0untFloyd** [comment]
As you noticed ffmpeg is needed for video swapping (images work without it). I'm no Mac owner though and can't help you with installing.

**Comment by Oil3** [comment]
**We need the error messages from terminal.**
Without it, I can only guess, and I guess: ffmpeg doesn't exist or python  cannot find it

Homebrew is very, _very good_ it is nothing like the likes of conda and redondant-diskspaceeating-pyenvn kind of stuff.
ffmpeg you dont install with pip.  
download homebrew, install it, relaunch terminal, install ffmpeg writing and return key _brew install ffmpeg_ relaunch terminal and then just do it.

so to recap:
give the error message from terminal
need to know:
python version
what "path" is when in terminal you  write and do return "env"
what is written when in terminal you write and do return "where ffmpeg"
what is written when in terminal you write and do return "where python3"

**Comment by Bir-Inder** [comment]
fixed with following install command: sudo apt install ffmpeg
